ui/components/app/dropdowns/network-dropdown.js

```diff
@@ -294,20 +303,26 @@ class NetworkDropdown extends Component {
         }}
         containerClassName="network-droppo"
         zIndex={55}
-        style={{
-          position: 'absolute',
-          top: '58px',
-          width: '309px',
-          zIndex: '55px',
-        }}
+        style={
+          this.props.dropdownStyles || {
+            position: 'absolute',
+            top: '58px',
+            width: '309px',
+            zIndex: '55',
+          }
+        }
         innerStyle={{
           padding: '16px 0',
         }}
       >
         <div className="network-dropdown-header">
-          <div className="network-dropdown-title">{t('networks')}</div>
-          <div className="network-dropdown-divider" />
-          {showTestnetMessageInDropdown ? (
+          {hideElementsForOnboarding ? null : (
+            <div className="network-dropdown-title">{t('networks')}</div>
+          )}
```

Comment:
May be `hideElementsForOnboarding && ...`

Reply:
`&&` has side effects like https://davidwalsh.name/fix-your-jsx , so I prefer to avoid using it.